Gemfile

```diff
@@ -54,7 +62,10 @@ group :development do
 
   # Spring speeds up development by keeping your application running in the background. Read more: https://github.com/rails/spring
   gem 'spring'
+  gem 'pry', '~> 0.10.4'
```

Comment:
That's what I mean. I don't like that it overwrites 'rails c'. I'd rather only be in pry when I explicitly call into it, personally